userdocs/src/usage/managing-nodegroups.md

```diff
@@ -76,6 +76,11 @@ nodeGroups:
     classicLoadBalancerNames:
       - dev-clb-1
       - dev-clb-2
+    asgMetricsCollection:
+      - granularity: 1Minute
+        metrics:
+          - GroupMinSize
```

Comment:
same as above, just a reference and docs

- GroupMinSize
- GroupMaxSize
- GroupDesiredCapacity
- GroupInServiceInstances
- GroupPendingInstances
- GroupStandbyInstances
- GroupTerminatingInstances
- GroupTotalInstances